tao_compiler/mlir/disc/IR/disc_ral_ops.td

```diff
@@ -29,7 +29,6 @@ def RAL_Dialect : Dialect {
 
 class RAL_Op<string mnemonic, list<Trait> traits> :
     Op<RAL_Dialect, mnemonic, traits> {
-  let verifier = [{ return Verify(*this); }];
```

Comment:
Why do we remote this check here?

Reply:
Because previous verify function does nothing but returning an OK directly.